CMakeLists.txt

```diff
@@ -258,6 +258,7 @@ endif()
 
 option(WITH_LIBSODIUM "Use libsodium instead of built-in tweetnacl" ON)
 option(WITH_LIBSODIUM_STATIC "Use static libsodium library" OFF)
+option(ENABLE_LIBSODIUM_RANDOMBYTES_CLOSE "Automatically closing libsodium randombytes (not threadsafe)" ON)
```

Comment:
If you want to list the explanation, please be precise "not threadsafe without getrandom()"